src/views/MapView.vue

```diff
@@ -36,9 +44,38 @@ onMounted(() => {
             <!-- we place the drawing module here so that it can receive the OpenLayers map instance through provide/inject -->
             <DrawingModule v-if="loadDrawingModule" />
             <!-- Needed to be able to set an overlay when hovering over the profile with the mouse -->
```

Comment:
I think this comment is now irrelevant, it could be deleted

It might make sense to explain a bit some part of what you do bellow, for instance why app version and copyright are removed on mobile (they are moved to the menu), etc...

Reply:
done